content/develop/connect/clients/_index.md

```diff
@@ -15,20 +15,29 @@ title: Connect with Redis clients
 weight: 45
 ---
 
-Here, you will learn how to connect your application to a Redis database. If you're new to Redis, you might first want to [install Redis with Redis Stack and Redis Insight]({{< relref "/operate/oss_and_stack/install/install-stack/" >}}).
-
-For more Redis topics, see [Using]({{< relref "/develop/use/" >}}) and [Managing]({{< relref "/operate/oss_and_stack/management/" >}}) Redis.
-
-<!--If you're ready to get started, see the following guides for the official client libraries you can use with Redis. For a complete list of community-driven clients, see [Clients](/resources/clients/).-->
-
-
-## High-level client libraries
-
-The Redis OM client libraries let you use the document modeling, indexing, and querying capabilities of Redis Stack much like the way you'd use an [ORM](https://en.wikipedia.org/wiki/Object%E2%80%93relational_mapping). The following Redis OM libraries support Redis Stack:
-
-* [Redis OM .NET]({{< relref "/integrate/redisom-for-net" >}})
-* [Redis OM Node]({{< relref "/integrate/redisom-for-node-js/" >}})
-* [Redis OM Python]({{< relref "/integrate/redisom-for-python/" >}})
-* [Redis OM Spring]({{< relref "/integrate/redisom-for-java/" >}})
-
-<hr>
\ No newline at end of file
+Use the Redis client libraries to connect to Redis servers from
+your own code. We currently support and document client libraries
+for 4 main languages:
```

Comment:
Two style guide violations: (1) "...currently...", see [here](https://developers.google.com/style/word-list#currently); (2) change "4" to "four", see [here](https://developers.google.com/style/numbers).

```suggestion
your own code. We support and document client libraries
for four main languages:
```

Reply:
Done, and there was another "currently" elsewhere.